.rhdh/docs/airgap.adoc

```diff
@@ -1,106 +1,199 @@
+A disconnected or restricted environment is an environment that does not have full access to the internet. So the cluster will not be able to pull any images from public registries like the Red Hat Ecosystem Catalog.
 
-On an OpenShift cluster operating in a restricted network, public resources are not available.
-However, deploying the RHDH Operator and running RHDH requires the following public resources:
+Deploying the RHDH Operator and running RHDH requires the following public resources:
 
 * Operator images (bundle, operator, catalog)
 * Operands images (RHDH, PostgreSQL)
 
-To make these resources available, we will need to replace them with their copies in a mirror registry accessible by the OpenShift cluster.
+To make these resources available, you will need to replace them with their copies in a mirror registry accessible by the cluster.
 
-We provide a helper script that mirrors all the necessary images and does the necessary plumbing to ensure those images will be used when installing the RHDH Operator and creating RHDH instances.
+We provide a helper script that mirrors all the necessary images and does all the heavy lifting to ensure those images will be used when installing the RHDH Operator and creating RHDH instances.
 
-This requires a target mirror registry, which you should already have if your OpenShift cluster is already ready to operate on a restricted network.
+This requires a target mirror registry, which you should already have if your cluster is already operating on a restricted network. If not and if you have an OpenShift cluster, you may want to expose and leverage the internal cluster registry.
 
-However, if you are preparing your cluster for disconnected usage, the script can also deploy a mirror registry in the cluster and use it for the mirroring process.
+Note that you can mirror the images to a mirror registry in a link:#_partially_disconnected_environments[partially disconnected environment] or in a link:#_fully_disconnected_environments[fully disconnected environment].
+
+Below are the prerequisites regardless of the target disconnected environment.
 
 *Prerequisites*
 
-* An active `oc` session with administrative permissions to the OpenShift cluster. See link:https://docs.openshift.com/container-platform/4.14/cli_reference/openshift_cli/getting-started-cli.html[Getting started with the OpenShift CLI].
-* An active `oc registry` session to the `registry.redhat.io` Red Hat Ecosystem Catalog. See link:https://access.redhat.com/RegistryAuthentication[Red Hat Container Registry Authentication].
```

Comment:
Active session to `registry.redhat.io` is still needed, at least with OCP. 
```
2025/02/05 15:08:57 render reference "registry.redhat.io/redhat/redhat-operator-index:v4.17": error resolving name for image ref registry.redhat.io/redhat/redhat-operator-index:v4.17: failed to authorize: failed to fetch anonymous token: unexpected status from GET request to https://registry.redhat.io/auth/realms/rhcc/protocol/redhat-docker-v2/auth?scope=repository%3Aredhat%2Fredhat-operator-index%3Apull&service=docker-registry: 401 Unauthorized
```

Reply:
Ah right, good catch. I've added this under the relevant additional prerequisites for each kind of environment: partially disconnected envs / fully disconnected envs (mirroring to disk).
Thanks.